packages/theme/css/characteristics/actionable.css

```diff
@@ -3,9 +3,11 @@
   --salt-actionable-cursor-active: pointer;
   --salt-actionable-cursor-disabled: not-allowed;
 
-  --salt-actionable-letterSpacing: 0.6px;
   --salt-actionable-textAlign: center;
```

Comment:
this one i assume should move too, to -text, if not done in design then they may have missed it

Reply:
good spot, lemme check with @pseys

Reply:
updated to include this change